flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/aggregate/GroupAggFunction.java

```diff
@@ -83,61 +85,57 @@
 	// stores the accumulators
 	private transient ValueState<RowData> accState = null;
 
+	private final StateTtlConfig ttlConfig;
+
 	/**
 	 * Creates a {@link GroupAggFunction}.
 	 *
-	 * @param minRetentionTime minimal state idle retention time.
-	 * @param maxRetentionTime maximal state idle retention time.
 	 * @param genAggsHandler The code generated function used to handle aggregates.
 	 * @param genRecordEqualiser The code generated equaliser used to equal RowData.
 	 * @param accTypes The accumulator types.
 	 * @param indexOfCountStar The index of COUNT(*) in the aggregates.
 	 *                          -1 when the input doesn't contain COUNT(*), i.e. doesn't contain retraction messages.
 	 *                          We make sure there is a COUNT(*) if input stream contains retraction.
 	 * @param generateUpdateBefore Whether this operator will generate UPDATE_BEFORE messages.
+	 * @param minRetentionTime minimal state idle retention time which unit is MILLISECONDS.
 	 */
 	public GroupAggFunction(
-			long minRetentionTime,
-			long maxRetentionTime,
 			GeneratedAggsHandleFunction genAggsHandler,
 			GeneratedRecordEqualiser genRecordEqualiser,
 			LogicalType[] accTypes,
 			int indexOfCountStar,
-			boolean generateUpdateBefore) {
-		super(minRetentionTime, maxRetentionTime);
+			boolean generateUpdateBefore,
+			long minRetentionTime) {
```

Comment:
nit: could you refactor the existing `minRetentionTime` to `stateRetentionTime`?

Reply:
OK